netci.groovy

```diff
@@ -2183,9 +2308,26 @@ Constants.allScenarios.each { scenario ->
                                     return
                                 }
                                 break
-                            case 'default':
+                            case 'normal':
                                 // Nothing skipped
                                 break
+                            case 'innerloop':
+                                def validOsPrTriggerArchConfigs = Constants.prTriggeredValidInnerLoopCombos[os]
```

Comment:
I think you should make this a function, e.g.: `isValidPrTriggeredInnerLoopJob(os, architecture, configuration)`